Formula/kat.rb

```diff
@@ -14,31 +14,53 @@ class Kat < Formula
 
   head do
     url "https://github.com/TGAC/KAT.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
   end
 
   needs :cxx11
 
   depends_on "pkg-config" => :build
-  depends_on "boost"
-  depends_on "python3"
+  depends_on "numpy"
+  depends_on "scipy"
+  depends_on "python"
```

Comment:
Please put these non-build dependencies in alphabetical order.

Reply:
will do.